include/fmt/format-inl.h

```diff
@@ -23,16 +23,14 @@
 #endif
 
 #ifdef _WIN32
-#  if defined(FMT_BIG_WIN) || defined(NOMINMAX)
-#    include <windows.h>
-#  else
-#    define NOMINMAX
-#    define WIN32_LEAN_AND_MEAN
-#    include <windows.h>
-#    undef WIN32_LEAN_AND_MEAN
-#    undef NOMINMAX
-#  endif
 #  include <io.h>
+FMT_BEGIN_NAMESPACE
```

Comment:
This is not needed.

Reply:
This is needed. See here: https://godbolt.org/z/nPG544

Reply:
But why is it overloaded? Shouldn't the types match?

Reply:
On Windows (x86, x64) DWORD is a 32 Bit unsigned integer. But in Windows.h it is defined as unsigned long / unsigned int. If we would use uint32 and include "windows.h" we would get an "overload extern C" error.

Reply:
A namespace helps just partially, it allows only C++ overloads for that function.